doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -128,7 +128,7 @@ Automatic declaration of schemes
 
 .. warning::
 
-   You have to be careful with this option since Coq may now reject well-defined
+   You have to be careful with this flag since Coq may now reject well-defined
```

Comment:
```suggestion
   You have to be careful with these flags since Coq may now reject well-defined
```

Would be better to indent the `.. warning::` block within the `.. flag::` block to better indicate the scope of the warning.